<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>315</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>315</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>315</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Photo Organizer v3</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout_source">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="1" column="2">
       <widget class="QPushButton" name="btnBrowseSource">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Browse‚Ä¶</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLineEdit" name="lineEditSource">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>26</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QLabel" name="labelSource">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;üìÇ&lt;/span&gt; Source Folder:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="buddy">
         <cstring>btnBrowseSource</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_Destination">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="1" column="2">
       <widget class="QPushButton" name="btnBrowseDestination">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Browse‚Ä¶</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLineEdit" name="lineEditDestination">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>26</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QLabel" name="labelDestination">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;üìÅ&lt;/span&gt; Destination Folder:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="buddy">
         <cstring>btnBrowseDestination</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBoxOptions">
      <property name="minimumSize">
       <size>
        <width>578</width>
        <height>99</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>99</height>
       </size>
      </property>
      <property name="title">
       <string>Sorting Options</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <layout class="QVBoxLayout" name="optionsLayout">
       <item>
        <widget class="QCheckBox" name="checkBoxSortByDay">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string>üìÖ Sort into subfolders by day</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxRemoveEmpty">
         <property name="text">
          <string>üóëÔ∏è Remove empty folders after sorting</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnStartSorting">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>25</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">
       QPushButton {
           background-color: #4CAF50;
           color: white;
           font-weight: bold;
           border-radius: 6px;
       }
       QPushButton:hover {
           background-color: #45A049;
       }
      </string>
      </property>
      <property name="text">
       <string>üöÄ Start Sorting</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>btnBrowseSource</tabstop>
  <tabstop>btnBrowseDestination</tabstop>
  <tabstop>checkBoxSortByDay</tabstop>
  <tabstop>checkBoxRemoveEmpty</tabstop>
  <tabstop>btnStartSorting</tabstop>
  <tabstop>lineEditSource</tabstop>
  <tabstop>lineEditDestination</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
